### PR TITLE
Prevent release and pre-release at the same time

### PR DESCRIPTION
**Pull request by bobleesj**
Same as the title.

### CODE REVIEW

File: release.py [hunk 1]
```diff
@@ -415,15 +415,19 @@ def cf_hash(opts, pargs):
...
     (opts, pargs) = parser.parse_args()
-    
+
     if len(pargs) < 2:
         parser.error("Improper usage. Too few arguments!")
     if len(pargs) > 2:
         parser.error("Improper usage. Too many arguments!")
-    
+
+    # Prevent release and pre_release both being true
```
Comment (sbillinge) [inline release.py:424]: please remove this line

File: release.py [hunk 2]
```diff
@@ -415,15 +415,19 @@ def cf_hash(opts, pargs):
...
+    if opts.release and opts.pre_release:
+        parser.error("Cannot release and pre-release at the same time.")
```
Comment (sbillinge) [inline release.py:426]: in general I like "reason for error. what to do to fix it" as a recipe that is most useful for users.  So adding "please rerun specifying one or the other" would work.  Here it is kind of obvious so not strictly necessary, but just good to get into the habit.  When error messages are aimed at a user, try and put yourself in the mind of the user, who in general is not a developer so avoid jargon.   Just some general comments there.....what you did was really great.

**Review by sbillinge — COMMENTED** [review]
please see comment.

File: release.py (hunk 2 above)
Comment (bobleesj) [reply release.py]: Yes. I will write error messages from the perspective of "reason for error. what to do to fix it".

File: release.py [hunk 3]
```diff
@@ -415,15 +415,18 @@ def cf_hash(opts, pargs):
...
-    
+
     if len(pargs) < 2:
         parser.error("Improper usage. Too few arguments!")
     if len(pargs) > 2:
         parser.error("Improper usage. Too many arguments!")
-    
+
+    if opts.release and opts.pre_release:
+        parser.error("Please re-run the command specifying either release or pre_release.")
```
Comment (bobleesj) [inline release.py:425]: "reason for error" -> user entered both release and pre_release"
" what to do to fix it" -> re-run

File: release.py [hunk 4]
```diff
@@ -421,9 +421,8 @@ def cf_hash(opts, pargs):
     if len(pargs) > 2:
         parser.error("Improper usage. Too many arguments!")
 
-    # Prevent release and pre_release both being true
     if opts.release and opts.pre_release:
-        parser.error("Cannot release and pre-release at the same time.")
+        parser.error("Please re-run the command specifying either release or pre_release.")
```
Comment (sbillinge) [inline release.py:425]: This is missing the error. It only has what to do about it. We want the mesage to say what went wrong then how to fix it.

**Review by sbillinge — COMMENTED** [review]
I stiill prefer the message to follow the model, see inline comment.

**Pull request merged**